py/pylya/cf.py

```diff
@@ -93,19 +94,20 @@ def cf(pix):
 def fast_cf(z1,r1,w1,d1,z2,r2,w2,d2,ang,same_half_plate):
     wd1 = d1*w1
     wd2 = d2*w2
-    rp = abs(r1-r2[:,None])*sp.cos(ang/2)
+    if rp_min<0 : rp = (r1-r2[:,None])*sp.cos(ang/2)
```

Comment:
if you have  rp_min>0 and you don't want to take the abs() there will be a problem since the rp grid will be positive. I'va done that to have the same results as before if rp_min>0. 
What do you think I should do ?